tests/pcap/src/handshake_message.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::*;
+use rtshark::Packet;
+use rtshark::RTSharkBuilder;
+use std::collections::HashSet;
+
+use crate::capture::*;
+
+#[derive(Debug, Clone, Default)]
+pub struct Message {
```

Comment:
Maybe this is just because I haven't spent as much time with Wireshark, but what _is_ a message? Like a ClientHello? So one of [these](https://www.rfc-editor.org/rfc/rfc8446#section-4.1) or a post-handshake message like SessionTicket? I think a short doc comment explaning the semantic meaning of this would be helpful.

Reply:
It's a TLS concept, not a Wireshark concept. I switched to more standard / less lazy naming and renamed it "HandshakeMessage". But I don't think our tests should really need to explain how TLS works, so the most I think comments should do is link to the RFC.